frontend/mission-components/package.json

```
@@ -25,21 +58,23 @@
     "ws": "^8.19.0"
   },
   "devDependencies": {
+    "@lhci/cli": "^0.13.0",
```

Biome (2.1.2)

[error] The key @lhci/cli was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)